src/transformers/models/llama/modeling_llama.py

```diff
@@ -68,6 +74,8 @@
 logger = logging.get_logger(__name__)
 
 _CONFIG_FOR_DOC = "LlamaConfig"
+_CHECKPOINT_FOR_DOC = "TinyLlama/TinyLlama-1.1B-Chat-v0.3"
```

Comment:
if this model was not trained on this task we should just not use one

Reply:
Then which model should be used for this purpose .

Reply:
No models if no models were trained